doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,37 +57,45 @@ Values are provided in kilobytes (KB).
-      +-------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Thermostat <matter_thermostat_sample>` (Debug)              |            28 |               745 |              4 |         32 |         809 |                             167 |
-      +-------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Thermostat <matter_thermostat_sample>` (Release)            |            28 |               639 |              4 |         32 |         703 |                             161 |
-      +-------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Debug)    |            28 |               730 |              4 |         32 |         794 |                             167 |
-      +-------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
-      | :ref:`Window Covering <matter_window_covering_sample>` (Release)  |            28 |               630 |              4 |         32 |         694 |                             161 |
-      +-------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | Sample                                                             |   MCUBoot ROM |   Application ROM |   Factory data |   Settings |   Total ROM |   Total RAM (incl. static HEAP) |
+      +====================================================================+===============+===================+================+============+=============+=================================+
+      | :ref:`Light Bulb <matter_light_bulb_sample>` (Debug)               |            28 |               794 |              4 |         32 |         858 |                             185 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Light Bulb <matter_light_bulb_sample>` (Lto debug)           |            28 |               732 |              4 |         32 |         796 |                             189 |
```

Comment:
```suggestion
      | :ref:`Light Bulb <matter_light_bulb_sample>` (LTO, Debug)           |            28 |               732 |              4 |         32 |         796 |                             189 |
```

`Debug` and `Release` are established terms, and LTO is just an additional build feature